content/en/docs/concepts/storage/volumes.md

```diff
@@ -679,7 +678,7 @@ PersistentVolume `nodeAffinity` is required when using local volumes. It enables
 the Kubernetes scheduler to correctly schedule Pods using local volumes to the
 correct node.
 
- PersistentVolume `volumeMode` can be set to "Block" (instead of the default
+PersistentVolume `volumeMode` can be set to "Block" (instead of the default
```

Comment:
@dulltz , What do you think about removing the sentence (end of line 682-683),
The `volumeMode` field requires `BlockVolume` feature gate to be enabled.

Reply:
@kbhawkey I agree with it!